task/inspect-image/0.1/inspect-image.yaml

```diff
@@ -10,7 +10,7 @@ metadata:
   name: inspect-image
 spec:
   description: >-
-    Get manifest data for source and base images and save them to workspace.
+    Uses Skopeo to inspect and analyze manifest data of container's source image, and its base image if available.
```

Comment:
Suggestion: Uses Skopeo to inspect and analyze the manifest data of the source image of a container, and its underlying image, if available.

Question: 
1. I'm sure devs know what is Manifest data. But do we need to simplify it for other users?
2. I've used `underlying` instead of `base`, did it change the mean we want to convey?

Reply:
> Suggestion: Uses Skopeo to inspect and analyze the manifest data of the source image of a container, and its underlying image, if available.

I specifically removed all the articles ("the, "an," "a"...) from all the descriptions based on this comment (https://github.com/redhat-appstudio/build-definitions/pull/331#discussion_r1138663720) by Jocelyn. Wdyt?

Reply:
If there is a precedence, please ignore my comment :) 
cc @jseseCCS

Reply:
BTW, my suggestion if we remove articles:  Inspect and analyze manifest data of the container's source image, and its base image (if available) with Skopeo.

Reply:
Thanks!

Reply:
> I'm sure devs know what is Manifest data. But do we need to simplify it for other users?

Sure, good idea. How about adding a second line:

Suggestion: An image's manifest data contains information about the layers that make up the image, the platforms for which the image is intended, and other metadata about the image.


> I've used underlying instead of base, did it change the meaning we want to convey?

Yes, the base image is a more recognized term.

Reply:
Thank you for the information. I think what you suggested is good. However, just for readability of you want you can break your suggestion into two sentences. 

The manifest data of an image contains information about the layers that make up the image and the platforms for which it is intended. It also includes other metadata about the image.

This is just a suggestion, of you want you can go ahead with what you suggested, since that is equally awesome :)